client/state/data-layer/wpcom/sites/atomic/transfers/latest/index.js

```diff
@@ -8,20 +9,20 @@ const requestTransferToAtomicStatus = ( action ) =>
 		{
 			apiNamespace: 'wpcom/v2',
 			method: 'GET',
-			path: `/sites/${ action.siteId }/atomic/transfers/`,
+			path: `/sites/${ action.siteId }/atomic/transfers/latest`,
 		},
 		action
 	);
 
-export const receiveError = ( error ) => {
-	return error;
-};
+const receiveError = ( action, error ) => [
+	setAtomicTransferStatus( action.siteId, action.softwareSet, error.error ),
+];
 
-export const receiveResponse = ( action, { success } ) => {
-	return success;
-};
+const receiveResponse = ( action, response ) => [
+	setAtomicTransferStatus( action.siteId, action.softwareSet, response.status ),
+];
```

Comment:
This needs reworking to store the latest transfer itself. We only need the status but I think where I've moved this to it makes more sense to make this about the whole transfer itself.